src/lib/types.ts

```diff
@@ -85,10 +85,25 @@ const postgresFunctionSchema = Type.Object({
     Type.Literal('VOLATILE'),
   ]),
   security_definer: Type.Boolean(),
-  config_params: Type.Union([Type.Dict(Type.String()), Type.Null()]),
+  config_params: Type.Union([Type.Record(Type.String(), Type.String()), Type.Null()]),
```

Comment:
@soedirgo version bumped `@sinclair/typebox` to the latest to use the new `Type.Record` (`Type.Dict` has been deprecated).